components/profile/ProfileEditor.js

```diff
@@ -197,6 +197,13 @@ export default function ProfileEditor({
         'You must enter position, institution, and domain information for each entry in your education and career history'
       )
     }
+    if (!profileContent.history.some((p) => !p.end || p.end >= new Date().getFullYear())) {
+      return promptInvalidValue(
+        'history',
+        profile.history?.[0]?.key,
+        'You must enter at least one current history record that could be education or career info'
```

Comment:
it seems there is another error that says "Error: You must enter at least one position for your education and career history", should we change this too?

Reply:
education and career history cannot be empty?

Reply:
I feel like this error message doesn't convey that it needs to be a current position. I would say, "Your Education & Career History must include at least one current position."